src/get-started/index.md.njk

```diff
@@ -8,3 +8,7 @@ show_subnav: true
 This section will enable you to get up and running as quickly as possible. Find easy access to toolkits, guides and helpful information on how to get setup, to start designing at HM Land Registry.
 
 To make your service consistent with GOV.UK, you should start by checking what exists in the [GOV.UK Design System](https://design-system.service.gov.uk).
+
```

Comment:
Can we add "Refer to the prototyping and developing setup guides." here please?

Reply:
I know the "prototyping" in this refers to https://hmlr-design-fix-duplica-zhyepi.herokuapp.com/get-started/prototyping/ but what does the "developing setup guides" refer to?

Reply:
Apologies, it should say "Refer to the prototyping and production setup guides."

Reply:
Changed